src/promptflow/promptflow/_sdk/_start_trace.py

```diff
@@ -41,6 +48,36 @@ def start_trace():
     print(f"You can view the trace from UI url: {ui_url}")
 
 
+def _start_pfs_in_background(pfs_port) -> None:
+    """Start a pfs process in background."""
+    if is_port_in_use(pfs_port):
+        _logger.debug("PFS is already serving on port %s", pfs_port)
```

Comment:
is it possible that another application uses this port? according to current code, if I start PFS with `--port` parameter, we will NOT record the port in config - maybe we can make port always record after it starts?

Reply:
1. update message here
2. add recording when set a pfs port and the port is not in use